data/json/items/generic.json

```diff
@@ -2754,5 +2758,36 @@
     "material": "plastic",
     "symbol": ")",
     "color": "green"
+  },
+  {
+    "type":"GENERIC",
+    "id" : "sheet_metal",
+    "name" : "sheet metal",
```

Comment:
what's wrong with the definition of sheet_metal in data/json/items/vehicle_parts.json?  Why is this repeated here?  I think changing the category makes sense, I think moving the definition around doesn't.